src/Services/Masa.Auth.Service.Admin/Infrastructure/EntityConfigurations/Subjects/UserEntityTypeConfiguration.cs

```diff
@@ -8,6 +8,8 @@ public class UserEntityTypeConfiguration : IEntityTypeConfiguration<User>
     public void Configure(EntityTypeBuilder<User> builder)
     {
         builder.HasKey(u => u.Id);
+        builder.Property(c => c.Account).UseCollation("SQL_Latin1_General_CP1_CS_AS");
```

Comment:
This is for SQL Server, if need to switch to mysql how find it and change it?

Reply:
mysql should replace 
`e.Property(p => p.CollationColumnFA).ForMySQLHasCollation("utf8_bin"); // defining collation in a property`
whether remove db limit ,change code logic limit?